### PR TITLE
Add types for vis.js

### PR DESCRIPTION
**Pull request by mmaitre314**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

File: vis/tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../tslint.json",
+  "rules": {
+    // Needed for ambient 'vis' namespace
```
Comment (andy-ms) [inline vis/tslint.json]: Try `export as namespace vis`.

**Comment by alexeagle** [comment]
ping @mmaitre314 I need this typing too - do you have time to finish the PR? Want help?

**Comment by andy-ms** [comment]
I'll merge this commit since my comments aren't really blocking. Although @alexeagle you could make a PR to fix `no-single-declare-module`.

**Comment by alexeagle** [comment]
Thanks Andy, will do

**Comment by andy-ms** [comment]
Hi @mmaitre314, in #13471 I've removed the `INetwork` and `ITimelineOptions` interfaces because they don't seem to be used, and there is already a `class Network` and `interface TimelineOptions`. If you'd like them in, please make a new PR (targeting `master`) adding them.

**Comment by mmaitre314** [comment]
Adding @MichaelBitard (the real author, I just repackaged the code) for info, but that sounds ok.

**Pull request merged**